integration/integration_test.go

```diff
@@ -34,6 +34,8 @@ import (
 
 var update = flag.Bool("update", false, "update golden files")
 
+const SPDXSchema = "https://raw.githubusercontent.com/spdx/spdx-spec/development/v2.3.1/schemas/spdx-schema.json"
```

Comment:
I'm sure we will forget to bump the version. Can we use spdx.Version somehow as it is used [here](https://github.com/aquasecurity/trivy/blob/7c68d4a7ecf4def588fe1dc3e6a40a4d1940dc75/pkg/sbom/spdx/marshal.go#L161)?

https://github.com/spdx/tools-golang/blob/6a271aaa7370038495ab7acdd0eab38e72666c9c/spdx/v2/v2_3/document.go#L15

I see a problem between v2.3 and v2.3.1. If it is hard to identify the patch version, I think it is ok to use v2.3.